constraints/miden-vm/bitwise.air

```diff
@@ -70,10 +77,28 @@ ev output_aggregation(main: [s, a, b, a_limb[4], b_limb[4], zp, z]):
         z = zp * 16 + a_xor_b when s
         z = zp * 16 + a_and_b when !s
 
+
+# Enforces the constraints for bitwise chiplet bus.
+ev bus_constraints([a, b, z], [b_chip]):
+    # u_i represents reduction of a, b abd z into a single value for row i.
+    let u_i = $alpha[1] * a + $alpha[2] * b + $alpha[3] * z
+
+    # m represents inversion of k1 periodic column.
+    let m = 1 - k1
```

Comment:
nit: it might be clearer to use a `binary_not` function like we do in other modules

Reply:
I was thinking that now we have `not` operator, we can avoid defining `binary_not` functions here and in other constraints.